Upstream-Status: Not Upstreamable

From fa23bfa1ca6ef105cb54dfe6a8e1ff09c287bfa8 Mon Sep 17 00:00:00 2001
From: Ovidiu-Adrian Vancea <ovidiu.vancea@ni.com>
Date: Thu, 1 Sep 2016 14:57:31 +0300
Subject: [PATCH] include: io.h needs inline def from compiler-gcc.h

io.h fails to compile without inline definition from compiler-gcc.h but will
complain if directly includes it. Use compiler.h which includes compiler-gcc.h

Signed-off-by: Ovidiu-Adrian Vancea <ovidiu.vancea@ni.com>
---
 arch/arm/include/asm/io.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/include/asm/io.h b/arch/arm/include/asm/io.h
index 1fbc531..144d9f4 100644
--- a/arch/arm/include/asm/io.h
+++ b/arch/arm/include/asm/io.h
@@ -28,6 +28,7 @@
 #if 0	/* XXX###XXX */
 #include <asm/arch/hardware.h>
 #endif	/* XXX###XXX */
+#include <linux/compiler.h>
 
 static inline void sync(void)
 {
-- 
2.7.4

